### DISCUSSION TITLE
Adding a vector store

### DISCUSSION THREAD

**Discussion by morgango** [langflow-ai/langflow#512] [General]
Hello.

I would like to add a vector store to the project, but am not exactly sure where to get started.  [Elasticsearch](https://python.langchain.com/docs/modules/data_connection/vectorstores/integrations/elasticsearch) is a supported vectorstore in LangChain, so I think it would be just a matter of making sure configuration information is passed through accurately.

Any hints on how/where to get started?

**Comment by moorthi07** [comment]
How to add AstraDB vector search into LangFlow?

**Comment by carlosrcoelho** [comment]
Hey!

Thanks for sharing your question here. I'm sorry for not getting back to you sooner.
Langflow has gone through some major changes since then. I recommend trying out the new version for the best experience. If you have any issues, please open a new one; we'll be here to help.
